components/proxy/src/main/java/com/hotels/styx/routing/config/RoutingObjectFactory.java

```diff
@@ -42,15 +43,36 @@
  * Builds a routing object based on its actual type.
  */
 public class RoutingObjectFactory {
-    private static final Map<String, HttpHandlerFactory> BUILTIN_HANDLERS = ImmutableMap.<String, HttpHandlerFactory>builder()
-            .put("StaticResponseHandler", new StaticResponseHandler.Factory())
-            .put("ConditionRouter", new ConditionRouter.Factory())
-            .put("InterceptorPipeline", new HttpInterceptorPipeline.Factory())
-            .put("ProxyToBackend", new ProxyToBackend.Factory())
-            .build();
+    public static final ImmutableMap<String, Schema.FieldValue> BUILTIN_HANDLER_SCHEMAS;
+
```

Comment:
These fields don't need to have blank lines between them.

Reply:
👍